### ISSUE TITLE
Try using spaCy text classifier

### ISSUE THREAD

**Issue by marco-c** [marco-c/bugbug#8]
We could try using the text classifier available in spaCy instead of gradient boosting and see how it behaves.

**Comment by ayush1999** [comment]
@marco-c What's the priority of this issue? Is it worth trying out the spacy model at this stage?

**Comment by marco-c** [comment]
Yes, definitely. It would require a bit of juggling around with the code, but it'd be useful to try.

**Comment by anirban-oss** [comment]
where do we want to use a spacy classifier? nlp.py?  spacy vectorizer is being used in a few files.
